BUILD.md

```diff
@@ -0,0 +1,137 @@
+# Hyperledger Cactus Build Instructions
+
+This is the place to start if you want to give Cactus a spin on your local
+machine or if you are planning on contributing.
+
+> This is not a guide for `using` Cactus for your projects that have business logic
```

Comment:
@sfuji822 

1. I added some clarification here so that users won't confuse this with a user-focused guide (which we don't yet have but should have one for sure).
2. Also updated the commit message and the PR title to avoid the same confusion as above mentioned. I can totally see how "getting started with the project" can be misleading without proper context (e.g. that it's getting started from a contributor's perspective, not a user's)

Reply:
@petermetz 
Can you modify the title with "Hyperledger Cactus SDK Building Instructions" ?

Reply:
@sfuji822 
Done.